axual-webclient/application_access_grant.go

```diff
@@ -75,3 +76,30 @@ func (c *Client) RevokeOrDenyGrant(applicationAccessGrantId string, reason strin
 	}
 	return nil
 }
+
+type ApplicationAccessGrantAttributes struct {
+	TopicId string
+	ApplicationId string
+	EnvironmentId string
+	AccessType string
+}
+func (c *Client) GetApplicationAccessGrantsByAttributes(data ApplicationAccessGrantAttributes) (*GetApplicationAccessGrantsByAttributeResponse, error) {
+	o := GetApplicationAccessGrantsByAttributeResponse{}
+	headers := map[string]string{
+		"Content-Type": "application/json",
+		"Accept":       "application/hal+json",
+	}
+
+	values := url.Values{}
+	values.Add("streamId", data.TopicId)
+	values.Add("applicationId", data.ApplicationId)
+	values.Add("environmentId", data.EnvironmentId)
+	values.Add("accessType", data.AccessType)
```

Comment:
This function should be generic enough to handle all valid scenarios of `GetApplicationAccessGrantsByAttributes` so all these attributes must be optional, and all the other possible attributes should be possible.